coralproject/talk
# [CORL-416] Disable Live Updates

## What does this PR do?

Allows the capability for moderators and admins to disable live updates site wide or per story.

## How do I test this PR?

Try disabling live updates via the environment variable, notice that all the options to configure the live updates disappear from the configuration panes.

Try disabling live updates site wide via the administration panel, notice how live updates no longer work on every story.

Try disabling live updates on a single story, notice how live updates no longer work on that story.

## Migration Required

```js
db.tenants.updateMany(
  {},
  {
    $set: {
      live: {
        enabled: true
      },
    },
  }
);
```
